source/includes/_android.md.erb

```diff
@@ -204,6 +204,29 @@ The `styles` option can be used to override default styles inside of Web Chat. T
 Setting custom styles based on Chat elements is inherently brittle, and can cause your styles to break over time. We do not recommend use of custom styles unless absolutely necessary. If custom styles are used, you should only rely on direct element IDs.
 </aside>
 
+#### zdChatterAuthCallback
+
+> When using AdaEmbedView
+
+```kotlin
+adaView.zdChatterAuthCallback = {
+    getToken() // this functions returns String
+}
+```
+
+> When using AdaEmbedDialog
+
+```kotlin
+val adaDialog = AdaEmbedDialog()
+adaDialog.zdChatterAuthCallback = {
+    getToken() // this functions returns String
+}
+```
+
+When called, the `zdChatterAuthCallback` can be used to request a JWT token from your API, and then have it passed to Ada. This allows for a shared trust between Ada and Zendesk, and in turn for verifiable chatter identity.
```

Comment:
```suggestion
When called, the `zdChatterAuthCallback` will request a JWT token (probably generated by your API), and then have it passed to Ada. This allows for validation between your App and Ada that the user is authenticated, and in turn between Ada and Zendesk for verifiable chatter identity.
```

Reply:
And we should probably add some reference to general documentation, which we will have ... eventually